query/src/pipelines/new/processors/transforms/hash_join/result_blocks.rs

```diff
@@ -121,6 +129,64 @@ impl JoinHashTable {
                     return Ok(vec![result]);
                 }
             }
+            Mark => {
+                let mut has_null = false;
+                // Check if there is any null in the probe block.
+                if let Some(validity) = input.column(0).validity().1 {
```

Comment:
What does the `column(0)` mean?

Reply:
The first column of subquery. 
For the SQL: `select * from t1 where t1.a in (select t2.a from t2); `  `t2.a` is the column(0).

Reply:
It looks confusing. Would you introduce a variable bound with it here and add some annotation about it?

Reply:
Ok